### PR TITLE
Added Methods Regarding Gaussian Noise

### PR DESCRIPTION
**Pull request by Inffzy**

### CODE REVIEW

File: tests/unit_tests/test_drone.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+"""
```
Comment (DarkmatterVale) [inline tests/unit_tests/test_drone.py]: You need to remove all of these.

**Review by DarkmatterVale — CHANGES_REQUESTED** [review]
Please add back the other unit tests and rebase so that you get the fix.

**Comment by Inffzy** [comment]
I added back the commented-out lines. But rebase made no change.

**Comment by DarkmatterVale** [comment]
You need to reinstall the SUASSystem package for the issue to be resolved on your system.

**Review by DarkmatterVale — APPROVED** [review]

**Pull request merged**